src/main/java/com/tsmms/skoop/user/GlobalUserPermissionScope.java

```diff
@@ -13,6 +13,10 @@
 	/**
 	 * Allows all users to see the owning user as a coach.
 	 */
-	FIND_AS_COACH
+	FIND_AS_COACH,
+	/**
+	 * Allows sales persons to use personalized profile (e.g. to pass it to customers).
+	 */
+	ALLOW_SALES_TO_USE_PERSONALIZED_PROFILE
```

Comment:
We have permissions with names `READ_USER_SKILLS` and `READ_USER_PROFILE`. So I would name it something like `READ_USER_PROFILE_BY_SALES`. But it's up to you

Reply:
I would like to avoid usage of the name like `READ_*` here since we grant to sales a permission **to use a profile outside** of SKOOP.
OK, let's leave it like that at the moment. Maybe we will find a better option in future.